cypress/integration/single-beacon-owner/i-can-register-a-beacon-with-many-uses.spec.ts

```diff
@@ -75,15 +85,13 @@ describe("As a single beacon owner with many uses,", () => {
     iCanSeeMyBeaconDetails();
     iCanSeeMyAdditionalBeaconInformation();
     iCanSeeASectionHeadingThatContains("Main use");
-    iCanSeeMyMaritimeUse(Purpose.PLEASURE);
+    iCanSeeMyLandUse();
     iCanSeeASectionHeadingThatContains("Second use");
-    iCanSeeMyAviationUse(Purpose.PLEASURE);
+    iCanSeeMyMaritimeUse(Purpose.PLEASURE);
     iCanSeeMyPersonalDetails();
     iCanSeeMyAddressDetails();
```

Comment:
Just missing an assertion here about seeing the Aviation use! 👀

Reply:
I will double check but believe this has already been updated!